lib/oxidized/manager.rb

```diff
@@ -8,7 +8,7 @@ class << self
       def load(dir, file)
         require File.join dir, file + '.rb'
         klass = nil
-        [Oxidized, Object].each do |mod|
+        [Oxidized, Source, Object].each do |mod|
```

Comment:
It helped me understanding the code better when using the full module path (`Oxidized::Source`) even if `Source` is  correct. What do you think?

Could you add a comment describing what this is doing, something like:
```
# Search the object in the different namespaces
# - Most objects are in the namespace Oxidized
# - Source objects have their own namespace (Oxidized::Source)
# - Models have no namespace (Object)
```

Reply:
I'm happy to take direction on using the full module path, I've updated the PR to use it and added your comment in.